### PR TITLE
Saas 16.4 3573252 oss and branch management bib

### PR DESCRIPTION
**Pull request by Levizar**
Prevent oss post init of the module
Prevent oss instantiation at company creation
Put oss tax on the root company

Task : 3573252

### CODE REVIEW

File: addons/l10n_eu_oss/tests/test_oss.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 class OssTemplateTestCase(AccountTestInvoicingCommon):
 
     @classmethod
-    def load_specific_chart_template(cls, chart_template_ref):
+    def setUpClass(cls, chart_template_ref):
```
Comment (william-andre) [inline addons/l10n_eu_oss/tests/test_oss.py:13]: Why was it not like this in the first place?

**Review by william-andre — COMMENTED** [review]
Awaiting functional validation

File: addons/l10n_eu_oss/tests/test_oss.py (hunk 1 above)
Comment (Levizar) [reply addons/l10n_eu_oss/tests/test_oss.py:13]: I don't remember, I think the reason is just "I'm dumb".
I fixed that fastly like a moron for a l10n nightly build and I got r+ed at the same speed.
I hadn't the opportunity to look it back and later on when I realized I hadn't a good reason to touch that file.

**Comment by Levizar** [comment]
@tsb-odoo

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by Levizar** [comment]
@william-andre I'm not sure what I'm supposed to do with this.
"The error is already known" but it seems to still be blocking the merge. :(

image

**Comment by william-andre** [comment]
It means that it is probably safe to just rebuild, also that mergebot can retry the test during the staging
I did it

**Comment by Levizar** [comment]
I already tried as it was the most obvious thing to do but I had the same result than you now. :/
I will try a last time in case it would work but I don't think so. 😔

**Comment by h4818** [comment]
@robodoo r+ rebase-ff

**Comment by h4818** [comment]
@robodoo retry

**Comment by william-andre** [comment]
This is a real error :D

**Comment by Levizar** [comment]
Fuck my life! 😂

**Comment by Levizar** [comment]
Should be ok now. I just forgot to provide the default parameter. :upside_down_face:

**Comment by william-andre** [comment]
@robodoo r+

**Comment by Levizar** [comment]
@william-andre thanks 🤩

**Pull request closed without merging**